nvc.1

```diff
@@ -501,6 +505,58 @@ hierarchy coverage report in HTML format, run:
 .Bd -literal -offset indent
 $nvc -c --merge=merged.covdb --report=<path_to_folder_for_html_report> first.covdb second.covdb third.covdb ...
 .Ed
+NVC supports following additional options to control coverage collection:
+.Bl -bullet
+.It
+.Cm count-from-undefined
+- When set, NVC also counts toggles
+.Cm U
+->
+.Cm 1
+as
+.Cm 0
+->
+.Cm 1
+and toggles
+.Cm U
```

Comment:
I would probably just write it plainly like `U -> 0`. I'm not sure the highlighting adds much.

Reply:
I would keep it like this. I see that in `man` in command line, its not really important, but in HTML format the highlighted version seems more readable.